src/sentry/notifications/utils/actions.py

```diff
@@ -33,12 +33,5 @@ class MessageAction:
 
 
 @dataclass
-class BlockKitMessageAction:
-    name: str
-    label: str
-    type: Literal["button", "select"] = "button"
-    url: str | None = None
-    value: str | None = None
-    action_id: str | None = None
+class BlockKitMessageAction(MessageAction):
```

Comment:
Other code was written in a way that suggests `BlockKitMessageAction` is supposed to have an is-a relationship to `MessageAction`. This change causes `BlockKitMessageAction` to gain a few attributes by inheritance, but only ones with default values, meaning no other code needs to change. (It's possible that some of those attributes aren't sensible, but if that's true then they should be moved from `MessageAction` into a different subclass).